### PR TITLE
Feat: Added Responsive Image Component

### PR DESCRIPTION
**Pull request by coldpigli**
Added responsive images, square images, round images of various sizes.

square-images
images-round

### CODE REVIEW

File: css/components/image.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.img-lg{
+    width: 300px;
+    height: 300px;
+}
+
+.resp-img{
+    max-inline-size: 100%;
+    width: 100%;
```
Comment (nishasen) [inline css/components/image.css:17]: To make your image responsive you can use max-width that works well

**Review by nishasen — COMMENTED** [review]
Please look into the comments

File: css/components/image.css (hunk 1 above)
Comment (coldpigli) [reply css/components/image.css:17]: Thanks. looking into it.

File: css/components/image.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    height: 250px;
+}
+
+.img-lg{
+    width: 300px;
+    height: 300px;
+}
+
+.resp-img{
+    max-inline-size: 100%;
```
Comment (vivekmehto) [inline css/components/image.css]: you already declared width and max-inline-size in reset css.

**Review by vivekmehto — COMMENTED** [review]
suggested one minor change apart from that everything seems perfect.

File: css/components/image.css (hunk 2 above)
Comment (coldpigli) [reply css/components/image.css]: Oh yes, just noticed. Thanks again.
Will make the changes.

**Pull request merged**